lib/tty/logger/config.rb

```diff
@@ -52,6 +55,7 @@ def initialize(**options)
         @formatter = options.fetch(:formatter) { :text }
         @date_format = options.fetch(:date_format) { "%F" }
         @time_format = options.fetch(:time_format) { "%T.%3N" }
+        @message_format = options.fetch(:message_format) { "%-25s" }
```

Comment:
Sorry about this! As the `date` and `time` formats are applicable to all handlers and the `message_format` is only applicable to `consol` handler, we should probably add it to the `console` handler instead.